jina/resources/k8s/template/deployment.yml

```diff
@@ -32,6 +32,7 @@ spec:
           args: {args}
           ports:
             - containerPort: {port}
+            - containerPort: {port-monitoring}
```

Comment:
if this is not yet handled, I would suggest to remove it for now

Reply:
my bad I thought that I had removed that part